translations/hu/get5.phrases.txt

```diff
@@ -212,6 +220,10 @@
     {
         "hu"            "Egy admin, szüneteli a mérkőzést."
```

Comment:
`Egy admin szüneteli a mérkőzést.`

Reply:
Elfogadva a javaslat. The suggestion is accepted